src/comments/comments.service.ts

```diff
@@ -14,7 +18,17 @@ export class CommentsService {
         return comments;
     }
 
-    async findOne(postId: number, commentId: number){
+    async findOne(postId: number, commentId: number, userId: number){
+        const comment = await this.prisma.comments.findUnique({
+            where:{
+                id: commentId,
+                postId
+            }
+        });
+        const action = await this.actionsService.addAction('Viewed',userId,'Comment',comment.id,comment)
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them